app/Models/Employee.php

```diff
@@ -2,10 +2,15 @@
 
 namespace App\Models;
 
-use Illuminate\Database\Eloquent\Factories\HasFactory;
+use App\Models\Wallet;
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
 
 class Employee extends Model
 {
     use HasFactory;
+    public function wallets()
```

Comment:
Take note of your naming conventions, hasOne means it's just one relationship. Kindly change this from wallets to wallet